sunpy/physics/differential_rotation.py

```diff
@@ -593,8 +593,13 @@ def differential_rotate(smap, observer=None, time=None, **diff_rot_kwargs):
     for key in all_keys:
         out_meta.pop(key)
 
+    if 'rsun_ref' not in out_meta:
+        warn_user(
+        f"'rsun_ref' not found in metadata. Using a default value {smap.rsun_meters} "
+        "If this value is incorrect, you will need to add the correct value to the map metadata."
+        )    
```

Comment:
```suggestion
```
No message should be emitted here because a logging message will already be emitted by the call to `.rsun_meters`.  Plus, `rsun_ref` is not the only way for the header to record the solar radius, so checking for its existence of just one keyword is not correct anyway.